Vagrantfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/focal64"
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL 
+    sudo apt-get update
+    sudo apt-get upgrade -y
+    sudo apt-get autoremove -y
+    sudo apt-get install -y python3 python3-pip python3-virtualenv
+    sudo ln -s /usr/bin/python3 /usr/bin/python
+    # below is important on Windows otherwise filesystem issues occur (e.g. "OError: [Errno 26] Text file busy")
+    cp -r /vagrant ~/todo
```

Comment:
This turned out to be critical for me as I'm running on Windows. I suspect it's not required if running Vagrant from a Unix-y filesystem (e.g. MacOS or Linux).

Reply:
The error is with sharing the virtual environment. So the alternative fix is to edit poetry.toml to no longer create the venv in the project folder.
If you actually wanted people to use Vagrant for a dev environment, then the poetry.toml fix is preferable so you can update the code frictionlessly.